src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -145,8 +148,8 @@
         // returns map of sobject name => list of fields
         var allFieldsBySObject = this.groupFieldsBySObject(everyField);
 
-        const opportunitySObjectName = "Opportunity";
-        const paymentSObjectName = "Payment";
+        const opportunitySObjectName = component.get('v.labels.opportunitySObjectName');
```

Comment:
This seems like one place we wouldn't want a label. We're talking about API names not object label, right?

Reply:
Yep I have fixed this just not pushed yet...